assets/js/alerts.js

```diff
@@ -234,3 +240,54 @@ function updateAll(alerts, currentFilters, isReset) {
   dropdownHandler.updateDropdowns(dataForDropdowns, currentFilters, isReset)
   updateURLParams(currentFilters)
 }
+
+// Slows fetch frequency and changes message when user interacting with drop downs
+let isDropDownOpen = false
+
+// uses event listeners for instant change in UI when drop down open
+document.addEventListener('mousedown', e => {
+  if (e.target.tagName.toLowerCase() === 'select') {
+    isDropDownOpen = true
+    alertsUpdateFrequencyMessage()
+  }
+})
+
+document.addEventListener('click', e => {
+  if (e.target.tagName.toLowerCase() !== 'select') {
+    isDropDownOpen = false
+    alertsUpdateFrequencyMessage()
+  }
+})
+
+function isAlertDropDownActive() {
+  return isDropDownOpen
+}
+
+// Refreshes data if stale - more than 30 seconds old
+let timer = 0
+
+function isDataThirtySecondsOld() {
+  if (timer >= 25) {
+    // next interval will tick at 30 seconds
+    return true
+  } else {
+    timer += 5
+    return false
+  }
+}
+
+// Message changes if drop down is open - from every 5 seconds to 30 seconds
+function alertsUpdateFrequencyMessage() {
+  try {
+    $('#alertsFetchStatus').empty()
+    if (isDropDownOpen) {
```

Comment:
You could just set a variable to be the text that changes and then just do 284/288 once inserting the variable into the string at the relevant place.
Also is there a reason you're using a strong tag in one and not the other?
Finally, although my suggestion requires the use of backticks, you don't need to use a backtick if it's just a plain string.